internal/pkg/manifest/svc.go

```diff
@@ -47,7 +47,10 @@ func (r *Range) IsEmpty() bool {
 // Parse extracts the min and max from RangeOpts
 func (r *Range) Parse() (min int, max int, err error) {
 	if r.Value != nil && !r.RangeConfig.IsEmpty() {
-		return 0, 0, errInvalidRangeOpts
+		return 0, 0, &errFieldMutualExclusive{
+			firstField:  "range",
+			secondField: "min/max",
```

Comment:
should this be "min/max/spot_from"?

Reply:
Also, I think `Range` is supposed to be a composite field that takes either a string, or a config. I'm not sure if this error is possible at all.

Reply:
Right right we should get rid of this.